status/actions.go

```diff
@@ -9,14 +11,42 @@ import (
 // MakeActionsFromWarnings returns a list of actions that can be performed on
 // the key to fix the warning.
 // Call `KeyAction.Enact(key)` to actually carry out the action.
-func MakeActionsFromWarnings(warnings []KeyWarning) []KeyAction {
-	now := time.Now()
+func MakeActionsFromWarnings(warnings []KeyWarning, args ...time.Time) []KeyAction {
```

Comment:
It's a neat trick to use `args` for time.Now(), but think we should specifically call it `now` and not be optional. Bit more obvs.